syntax ="proto3";
package IM;
message ModifyInfo
{
	int32 networkType=1;
	int32 msgID=2;//每条消息的标识
	int32 recvID=3;
	int32 sendID=4;
	bytes password=5;
	bytes nickname=6; //昵称
	bytes phone=7;
	bytes desc=8;   //介绍
	int32 sex=11;
	bytes icon=9;   //头像
	enum RequireType{  
		READ=0;
		WRITE=1;
	}
	RequireType requireType=10;
	int32 userID=12;
	int32 state=13;
}
